server/views/pages/barcode/edit-contact-details.njk

```diff
@@ -47,7 +47,6 @@
               name: "prisonerName",
               value: form.prisonerName,
               attributes: {
-                autofocus: '',
```

Comment:
Autofocus is a bad thing for accessibility - it drops users right in the middle of the page meaning they lose all context of where they are.